### ISSUE TITLE
[Screen readers: Designer_Card Elements]: Labels are not associated with the controls under "Elements Properties" pane.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6956]
https://microsoft.visualstudio.com/OS/_workitems/edit/30898371
